packages/engine/src/assets/functions/createGLTFLoader.ts

```diff
@@ -1,14 +1,26 @@
 import { VRMLoaderPlugin } from '@pixiv/three-vrm'
+import { REVISION } from 'three'
 
 import { isClient } from '../../common/functions/isClient'
+import { Engine } from '../../ecs/classes/Engine'
 import { GLTFHubsComponentsExtension } from '../classes/GLTFHubsComponentsExtension'
 import { GLTFHubsLightMapExtension } from '../classes/GLTFHubsLightMapExtension'
 import { GLTFInstancingExtension } from '../classes/GLTFInstancingExtension'
 import { GLTFRemoveMaterialsExtension } from '../classes/GLTFRemoveMaterialsExtension'
 import { DRACOLoader } from '../loaders/gltf/DRACOLoader'
 import { GLTFLoader } from '../loaders/gltf/GLTFLoader'
+import { KTX2Loader } from '../loaders/gltf/KTX2Loader'
+import { MeshoptDecoder } from '../loaders/gltf/meshopt_decoder.module'
 import { NodeDRACOLoader } from '../loaders/gltf/NodeDracoLoader'
 
+export const initializeKTX2Loader = (loader: GLTFLoader) => {
+  const ktxLoader: any = new KTX2Loader()
+  const THREE_PATH = `https://unpkg.com/three@0.${REVISION}.x`
+  ktxLoader.setTranscoderPath(`${THREE_PATH}/examples/js/libs/basis/`)
```

Comment:
we have the basis loader served from client public folder already, we don't want to be relying on external dependencies that we have no control over and potentially break

Reply:
The basis decoders are now being loaded from the public folders